config/jetpack-cloud-horizon.json

```diff
@@ -81,6 +81,7 @@
 		"jetpack-cloud-partner-portal": true,
 		"jetpack-cloud-plugin-management": true,
 		"jetpack-cloud-pricing": true,
+		"jetpack-cloud-manage-pricing": false,
```

Comment:
If we set this feature flag to true in Horizon, we could test it here in this PR using the Jetpack Cloud live link (like staging).

Reply:
Here: 
![image](https://github.com/Automattic/wp-calypso/assets/9832440/34ccd167-f2b6-4e68-8ea5-5c17a5d383bf)